### PR TITLE
[SQL] Add examples of Create Server and MI with TDE CMK

### PR DESCRIPTION
**Pull request by viparek**
## Description

Adding new examples for Create SQL Server and create managed instance with TDE CMK

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Sql/Sql/help/New-AzSqlInstance.md [hunk 1]
```diff
@@ -317,6 +317,34 @@ ZoneRedundant            : true
 
 This command creates a new instance with external administrator properties and azure active directory only authentication enabled.
 
+### Example 9: Create a new instance with TDE CMK
+```powershell
+New-AzSqlInstance -Name managedInstance1 -ResourceGroupName ResourceGroup01 -Location westcentralus -AdministratorCredential (Get-Credential) -SubnetId "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/resourcegroup01/providers/Microsoft.Network/virtualNetworks/vnet_name/subnets/subnet_name" -LicenseType LicenseIncluded -StorageSizeInGB 1024 -VCore 16 -SkuName GP_Gen4 -DnsZonePartner "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/resourcegroup01/providers/Microsoft.Sql/managedInstances/partnerServerForDnsZone" -AssignIdentity -IdentityType "UserAssigned" -PrimaryUserAssignedIdentityId "Primary UserManaged identity id" -UserAssignedIdentityId "UserManaged identity id" -KeyId "AzureKeyVault Uri"
```
Comment (strehan1993) [inline src/Sql/Sql/help/New-AzSqlInstance.md:322]: Let's add dummy resource ids for UserAssignedIdentityId and PrimaryUserAssignedIdentityId 
Let's add dummy AKV URLs for key id

**Review by strehan1993 — APPROVED** [review]

**Comment by viparek** [comment]
@isra-fel This is reviewed by the owning team and ready to merge. Please sign off and merge this PR.

**Review by isra-fel — APPROVED** [review]
If PR only modifies documents, changelog does not have to be updated.

**Review by ericshape — APPROVED** [review]
LGTM

**Pull request merged**